pkg/util/ansible_execution.go

```diff
@@ -55,6 +55,21 @@ func AnsibleExecution(
 	if err != nil && !k8serrors.IsNotFound(err) {
 		return err
 	}
+
+	// If the AnsibleEE already exists and the config has changed, then we will delete
+	// the existing AnsibleEE and re-create it below to roll out the new config.
+	if ansibleEE != nil && ansibleEE.Spec.Env[0].Value != aeeSpec.Env[0].Value {
+		helper.GetLogger().Info("Redeploying OpenStackAnsibleEE: %s", ansibleEE.Name)
+		err := helper.GetClient().Delete(ctx, ansibleEE)
```

Comment:
This also would delete the Job, which deletes the ansible execution logs. How do we account for saving those logs somewhere for debuggability?

Reply:
It does indeed delete the job. I guess the alternative would be to create a new instance of the AEE object with part of the configHash in the name. Although, that will break the rest of the logic above for checking and comparing the config hash in the existing AEE, but doable I think.

Could probably remove that logic above and just look for a AEE with part of the confighash in the name, if I get a `k8serrors.IsNotFound()` we know we need to create a new one.